content/arduino-cloud/01.getting-started/10.iot-cloud-micropython/content.md

```diff
@@ -0,0 +1,192 @@
+---
+title: 'Connecting to Arduino IoT Cloud using MicroPython'
+description: 'Learn how to use the MicroPython library to connect to the Arduino IoT Cloud and control an LED.'
+tags: 
+  - IoT
+  - MicroPython
+author: 'Sebastian Romero'
+libraries: 
+  - name: Arduino IoT Cloud Python
+    url: https://github.com/arduino/arduino-iot-cloud-py
+---
+
+## Introduction 
+
+This tutorial guides you on how to use the MicroPython library to connect your Arduino device to the Arduino IoT Cloud.
+
+## Goals
+
+The goals of this tutorial are:
+
+- Connect your Arduino device to your Wi-Fi network.
+- Connect your Arduino device to the Arduino IoT Cloud.
+- Control an LED using the Arduino IoT Cloud.
+
+## Hardware & Software Needed
+
+- [Arduino Lab for MicroPython](https://labs.arduino.cc/en/labs/micropython)
+- [Arduino GIGA R1 WiFi](/hardware/giga-r1-wifi) or [Portenta H7](/hardware/portenta-h7)
+- MicroPython >= 1.2 installed on your Arduino device (download firmware [here](/micropython/#firmware)).
+
+
+
+## Setting Up Your Device and IoT Cloud
+
+Before we start, make sure you have MicroPython installed on your board. If you haven't you can follow [this tutorial](https://docs.arduino.cc/micropython/basics/board-installation). Then configure a "thing" in the [Arduino IoT Cloud](https://create.arduino.cc/iot/) consisting of two boolean variables called `led` and `ledSwitch`. To set up a "thing" and a corresponding dashboard, please follow these two tutorials:
+
+- [IoT Cloud Variables](https://docs.arduino.cc/arduino-cloud/getting-started/cloud-variables)
+- [IoT Cloud Dashboards & Widgets](https://docs.arduino.cc/arduino-cloud/getting-started/dashboard-widgets)
+
+The resulting thing and dashboard should look similar to the following:
+
+![Thing with two boolean variables.](./assets/thing.png)
+
+![Dashboard with an LED and a Switch widget.](./assets/dashboard.png)
+
+Also, your device needs to be registered. Follow the flow "Any Device" ("Manual") when clicking "Add" in the "Devices" tab.
```

Comment:
Please use bold when you are highlighting button names (if that is what these in " " are)